_appmap/web_framework.py

```diff
@@ -233,6 +238,9 @@ def remote_enabled(self):
         """Return True if the AppMap middleware has enabled remote recording, False otherwise."""
 
     def run(self):
+        if not Env.current.enabled:
+            return
+
```

Comment:
This change addresses something different than error handling of application/json requests.  Is there, or should there be, a testcase that fails without this change?

Reply:
Reading this, I realized that my comment on [b24fcd9](https://github.com/getappmap/appmap-python/pull/211/commits/b24fcd9bb1b8df19a39055d233d83ca6dc18ad5e) didn't mention that the tests I restored were broken, and that some of the change address those failures. I'll fix that comment.

Having said that, it's possible that this change wasn't actually necessary,  I'll double-check.

Reply:
Ok, I took this change out and updated the comment.

Reply:
Thanks.  Make sure to push the updated branch.

Reply:
Done. (My original attempt to push it failed.)